### DISCUSSION TITLE
My ps2 with fmcb gets stuck if the blueretro is connected

### DISCUSSION THREAD

**Discussion by andreivlad9001** [darthcloud/BlueRetro#601] [Q&A] [answered]
I have a ps2 fat 50004 that has fmcb 1.95 installed on the memory card and a hdd.
When i turn it on the console freezes on the logo if the blueretro is connected and powered on with usb.
The blueretro made from a esp32 devkit e
With 2 ps2 controller ports. The installed version is 1.8.2 hw1.

**Comment by andreivlad9001** [comment, marked as answer]
Running power from the usb of the console solved the problem.

**Comment by delgoes** [comment]
I have the same problem but I couldn't solve it by connecting to the USB of the console, any suggestions?

**Reply by andreivlad9001** [reply]
Does it freeze if you power it separately?

**Reply by andreivlad9001** [reply]
You did not mention more details about your problem so i dont have any sugestions.

**Comment by CorrePlayer** [comment]
My ps2 also freezes when using ds4 via usb in burnout revenge
